cmd/system-probe/modules/processmonitor.go

```diff
@@ -0,0 +1,71 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+//go:build linux
+
+package modules
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/cmd/system-probe/config"
+	sysconfigtypes "github.com/DataDog/datadog-agent/cmd/system-probe/config/types"
+	pkgconfigsetup "github.com/DataDog/datadog-agent/pkg/config/setup"
+	"github.com/DataDog/datadog-agent/pkg/process/monitor"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+
+	"github.com/DataDog/datadog-agent/cmd/system-probe/api/module"
+)
+
+func isEventStreamEnabled() bool {
+	return pkgconfigsetup.SystemProbe().GetBool("service_monitoring_config.enable_event_stream")
+}
+
+// ProcessMonitor - Event monitor Factory
+var ProcessMonitor = module.Factory{
+	Name:             config.ProcessMonitorModule,
+	ConfigNamespaces: processMonitorModuleConfigNamespaces,
+	Fn:               createProcessMonitorModule,
+	NeedsEBPF: func() bool {
+		return isEventStreamEnabled()
+	},
+	IgnoreForSuccessCheck: true,
+}
+
+var processMonitorModuleConfigNamespaces = []string{"service_monitoring_config"}
+
+func createProcessMonitorModule(_ *sysconfigtypes.Config, _ module.FactoryDependencies) (module.Module, error) {
+	log.Infof("Initializing process monitor...")
```

Comment:
Do we need that log? the module runner should log that for every module

Reply:
You're right, removed.